docs-new/docs/graphql/core/api-reference/metadata-api/relationship.mdx

```diff
@@ -578,13 +534,7 @@ X-Hasura-Role: admin
 }
 ```
 
-<div class="note">
-
-<div class="title">
-
-Note
-
-</div>
+::info Note
```

Comment:
please add one more `:`

UI is failing to parse due to syntax issue

Reply:
Done